src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -64,7 +64,9 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NetCoreAppNativeLibrary Include="@(NetCoreAppNativeLibrary->'%(Identity)')">
         <EscapedPath>$(IlcFrameworkNativePath)lib%(Identity).a</EscapedPath>
       </NetCoreAppNativeLibrary>
+      <NativeLibrary Include="$(IntermediateOutputPath)/source_code/libs/System.Globalization.Native/build/libSystem.Globalization.Native-Static.a" Condition="'$(LocalSystemGlobalizationNative)' != ''"/>
```

Comment:
SystemGlobalizationNative is internal implementation. We want to user-facing property to be named in terms what users understand and that is serlf-explanatory, like `ICUStaticLinking`. (Feel free to come up with a better name.)

Reply:
More ideas: `StaticICU` , `StaticICULinking`, ... .